docs/specification.md

```diff
@@ -346,9 +346,9 @@ Note: in order to be able to perform a swap, this contract must be made an
 - When the `y` token is `ctez`, a [protocol fee](#fees) is subtracted
   after converting the `x` tokens to `ctez`.
 - If the swap is no longer acceptable because the `deadline` was not met, fails
-  with `TIMED_OUT`.
+  with `past_deadline_err` error code (Check [Error Codes](/docs/error-codes.md) for details).
```

Comment:
Instead of having a link to `error-codes.md` here, I think we should have it at the bottom of this document (see the `TODO` in the `# Errors` section).

```suggestion
  with `past_deadline_err` error code.
```

Reply:
Make sense, I've updated this.